blog/authors.yml

```diff
@@ -0,0 +1,23 @@
+afflom:
+  name: Alex Flom
+  title: Docusaurus maintainer
+  url: https://github.com/afflom
+  image_url: https://github.com/afflom.png
+
+jpower432:
+  name: Jennifer Power
+  title: Docusaurus maintainer
+  url: https://github.com/jpower432
+  image_url: https://github.com/jpower432.png
+
+sabre1041:
+  name: Andrew Block
+  title: Docusaurus maintainer
```

Comment:
Project Manager

Reply:
resolved in [7f31d4f](https://github.com/uor-framework/universalreference.io/pull/11/commits/7f31d4ffddba6c100941e3d67a841925fee8f5d1)